examples/Makefile

```diff
@@ -0,0 +1,14 @@
+WORK_DIR:=$(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+PROTO_HOME=/usr/local/opt/include
+SRC_DIR=$(WORK_DIR)
+TARGET_DIR=$(WORK_DIR)
+
+PROTOS := user.proto
+
+compile: $(PROTOS)
```

Comment:
nit: This target will always be rebuilt, a proper target should be:
`output_file: input_file`

But I guess it is fine.

Reply:
Resolving as "fine the way it is"

Reply:
Forgot this was the examples one not the tests Makefile. Fixed with proper target